frontend/app/src/pages/main/events/components/event-columns.tsx

```diff
@@ -78,6 +78,13 @@ export const columns = ({
         return <div>{relativeDate(row.original.metadata.createdAt)}</div>;
       },
     },
+    {
+      accessorKey: 'workflows',
+      header: () => <></>,
+      cell: () => {
+        return <div></div>;
+      },
+    },
```

Comment:
Not so proud of this part, when I started I figured it was the easiest way to add it, but not sure if we actually want to show workflow names as it could be a lot. So I just don't show them right now, but the column has to exist here, otherwise the filter won't work with the existing column filter logic.

Reply:
This seems fine, but would probably make sense to add a comment to the codebase so future maintainers are aware that it shouldn't be removed?